testing/integration-tests/Cargo.toml

```diff
@@ -15,6 +15,9 @@ description = "Subxt integration tests that rely on the Substrate binary"
 [features]
 default = ["subxt/integration-tests"]
 
+# Enable to run the tests with Light Client support.
+unstable-light-client = ["subxt/integration-tests", "subxt/unstable-light-client"]
```

Comment:
I did a grep and offhand can't see any reference to `integration-tests` nowadays, so I think this feature can just be removed from here and Subxt? (would be ideal; logic that is required specifically for testing is never desirable if it can be avoided :))